tests/ijby/test-func-unary.py

```diff
@@ -169,12 +170,64 @@ def test_dt_isna_joined():
                    D=[22, 33, 44, 55],
                    E=[True, False, True, False])
     JDT.key = 'A'
-    RES = DT[:, dt.isna(g[1:]), join(JDT)]
+    RES = DT[:, isna_fn(g[1:]), join(JDT)]
     frame_integrity_check(RES)
     assert RES.to_list() == [[True, True, False, True, False]] * 4
 
 
 
+@pytest.mark.parametrize("src", srcs_bool + srcs_int + srcs_float + srcs_str)
+def test_dt_isna_scalar(src):
+    for val in src:
+        assert dt.isna(val) == (val is None)
+
+
+@pytest.mark.parametrize("src", srcs_int + srcs_float)
+def test_dt_math_isna_scalar_not_implemented(src):
+    for val in src:
+        with pytest.raises(NotImplementedError,
+                           match = "This operation is not implemented yet"):
+            dt.math.isna(val)
+
+
+@pytest.mark.parametrize("src", [srcs_bool[0], srcs_str[0]])
+def test_dt_math_isna_scalar_wrong_arg(src):
+    for val in src:
+        with pytest.raises(TypeError,
```

Comment:
this is strange... `isna()` should be applicable to any type

Reply:
Well, I guess this is because of https://github.com/h2oai/datatable/blob/master/c/expr/expr_unaryop.cc#L288-L290 and we also never treat booleans here.